util/reggen/ip_block.py

```diff
@@ -20,6 +20,44 @@
 from reggen.signal import Signal
 from reggen.countermeasure import CounterMeasure
 
+# Known comportable IP names and associated CIP_IDs.
+KNOWN_CIP_IDS = {
```

Comment:
Do we think keeping this here is fine or should this be moved to a kind of index file (e.g. `hw/cip_ids.hjson`)?

Reply:
Spoke with @andreaskurth about this, and we think it's probably fine to keep this here for now.